docs/winrt-apis-for-desktop.md

```diff
@@ -0,0 +1,59 @@
+# WinRT API changes for Desktop apps
+
+In WinUI 3 Preview 4, we have made changes to the way Windowing works in Desktop apps, by no longer creating a CoreWindow or CoreApplicationView. As a result, many WinRT APIs will no longer be supported in Dekstop apps. This document will list out APIs that have been affected by this change, with replacement or workaround APIs that will allow you to achieve the same functionality. 
```

Comment:
Suggesting some more background, and pulling to the top that note that APIs that depend on unsupported classes also become unsupported.

```suggestion
Some Windows APIs are designed for and only completely supported in a UWP app, especially CoreDispatcher, CoreWindow, ApplicationView, and some related classes. These APIs are partially supported in Desktop apps when _hosting_ Xaml (Xaml Islands), and in WinUI3 previews before Preview4. As of WinUI3 Preview4 they are fully unsupported.

Note that when a class is unsupported in Desktop it can lead to other unsupported classes. For example, several classes in the Windows API have a static GetForCurrentView method, such as [UIViewSettings.GetForCurrentView](https://docs.microsoft.com/uwp/api/Windows.UI.ViewManagement.UIViewSettings.GetForCurrentView), which is a pattern that indicates a dependence on ApplicationView, and so are not supported in Desktop apps.

This document lists out these APIs, and suggests replacements where available to achieve the same functionality. 
```